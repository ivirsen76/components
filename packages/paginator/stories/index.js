import React from 'react'
import Paginator from '../src'
import { storiesOf } from '@kadira/storybook' // eslint-disable-line

const stories = storiesOf('paginator', module)

stories.add('first page', () => <Paginator total={50} />)

stories.add('middle page', () => <Paginator total={50} currentPage={12} />)

stories.add('last page', () => <Paginator total={50} currentPage={50} />)

stories.add('just a few pages', () => <Paginator total={3} />)
