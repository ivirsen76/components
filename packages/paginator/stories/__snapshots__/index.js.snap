// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`paginator first page 1`] = `
<div
  className="ui pagination tiny menu"
>
  <a
    className="disabled item"
    onClick={[Function]}
  >
    First
  </a>
  <a
    className="disabled item"
    onClick={[Function]}
  >
    «
  </a>
  <a
    className="active item"
    onClick={[Function]}
  >
    1
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    2
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    3
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    4
  </a>
  <div
    className="disabled item"
  >
    ...
  </div>
  <a
    className="item"
    onClick={[Function]}
  >
    48
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    49
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    50
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    »
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    Last
  </a>
</div>
`;

exports[`paginator just a few pages 1`] = `
<div
  className="ui pagination tiny menu"
>
  <a
    className="disabled item"
    onClick={[Function]}
  >
    First
  </a>
  <a
    className="disabled item"
    onClick={[Function]}
  >
    «
  </a>
  <a
    className="active item"
    onClick={[Function]}
  >
    1
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    2
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    3
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    »
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    Last
  </a>
</div>
`;

exports[`paginator last page 1`] = `
<div
  className="ui pagination tiny menu"
>
  <a
    className="item"
    onClick={[Function]}
  >
    First
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    «
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    1
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    2
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    3
  </a>
  <div
    className="disabled item"
  >
    ...
  </div>
  <a
    className="item"
    onClick={[Function]}
  >
    47
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    48
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    49
  </a>
  <a
    className="active item"
    onClick={[Function]}
  >
    50
  </a>
  <a
    className="disabled item"
    onClick={[Function]}
  >
    »
  </a>
  <a
    className="disabled item"
    onClick={[Function]}
  >
    Last
  </a>
</div>
`;

exports[`paginator middle page 1`] = `
<div
  className="ui pagination tiny menu"
>
  <a
    className="item"
    onClick={[Function]}
  >
    First
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    «
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    1
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    2
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    3
  </a>
  <div
    className="disabled item"
  >
    ...
  </div>
  <a
    className="item"
    onClick={[Function]}
  >
    9
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    10
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    11
  </a>
  <a
    className="active item"
    onClick={[Function]}
  >
    12
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    13
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    14
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    15
  </a>
  <div
    className="disabled item"
  >
    ...
  </div>
  <a
    className="item"
    onClick={[Function]}
  >
    48
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    49
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    50
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    »
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    Last
  </a>
</div>
`;
